types/react-google-tag-manager/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for react-google-tag-manager 2.2
+// Project: https://github.com/gaearon/react-google-tag-manager
```

Comment:
The project does not seem to exist at this location. According to npm, the project can be found at https://github.com/holidaycheck/react-google-tag-manager